dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship.unittests/Integration/GetApprenticeshipsAsProviderIntegrationTests.cs

```diff
@@ -31,8 +35,8 @@ namespace Dfc.ProviderPortal.FindAnApprenticeship.UnitTests.Integration
     public class GetApprenticeshipsAsProviderIntegrationTests
     {
         private readonly TelemetryClient _telemetryClient;
-        private readonly IAppCache _appCache;
-        private readonly Mock<IConfiguration> _configuration;
+        private readonly Mock<Func<DateTimeOffset>> _nowUtc;
```

Comment:
As above

Reply:
I prefer it as it makes manipulating the value returned within the tests easier.